docs/en/integrations/data-ingestion/clickpipes/index.md

```diff
@@ -73,39 +74,39 @@ import WarpStreamSVG from "../../images/logos/warpstream.svg";
 
   The summary table provides controls to display sample data from the source or the destination table in ClickHouse
 
-  ![View source](./images/cp_source.png)
-
   ![View destination](./images/cp_destination.png)
 
   As well as controls to remove the ClickPipe and display a summary of the ingest job.
 
   ![View overview](./images/cp_overview.png)
 
-9. **Congratulations!** you have successfully set up your first ClickPipe. This job will be continuously running, ingesting data in real-time from your remote data source.
+9. **Congratulations!** you have successfully set up your first ClickPipe. If this is a streaming ClickPipe it will be continuously running, ingesting data in real-time from your remote data source. Otherwise it will ingest the batch and complete.
```

Comment:
I don't see an explanation of `streaming ClickPipe`. Should we add a quick intro for continuous and one-off ingestion types?